src/test/java/com/epam/automation/inside.java

```diff
@@ -1,136 +0,0 @@
-package com.epam.automation;
-
-import org.openqa.selenium.By;
-import org.openqa.selenium.StaleElementReferenceException;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
-import org.openqa.selenium.firefox.FirefoxDriver;
-import org.testng.annotations.AfterMethod;
-import org.testng.annotations.BeforeMethod;
-
-import java.util.List;
-import java.util.concurrent.TimeUnit;
-
-import static org.testng.Assert.assertTrue;
-
-public class inside {
-    public WebDriver driver;
-
-    @BeforeMethod
-    public void openBrowser() {
-        System.setProperty("webdriver.gecko.driver", "drivers/geckodriver.exe");
-        driver = new FirefoxDriver();
-        driver.get("https://www.epam.com");
-        System.out.println(driver.getTitle());
-        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
-    }
-
-    public void navigation(String Menu) {
-        // menüpontokra tud lépni enumból kiszedett címekkel, almenüket jelenleg nem raktam bele
-        // az oldal sikeres betöltését azzal ellenőrzi, hogy megjelent e a bal felső sarokban lévő epam logó
-        driver.findElement(By.xpath("//a[@href='" + Menu + "']")).click();
```

Comment:
Try to avoid xpath as much as you can, it is relative in the dom and slow in your automation.
(Check Page Object pattern as well)